packages/gui/src/components/offers/OfferShareDialog.tsx

```diff
@@ -1481,7 +1481,7 @@ export default function OfferShareDialog(props: OfferShareDialogProps) {
       </DialogTitle>
 
       <DialogContent dividers>
-        <Flex flexDirection="column" gap={2}>
+        <Flex flexDirection="column" gap={2} paddingTop="16px">
```

Comment:
Can you use the number 2 instead of the string 16px? material-ui will replace it with a correct value, which is always multiplied by 8px

1 = 8px
1.5 = 12px
2 = 16px
...